user-workspace/src/App.overrides.tsx

```
@@ -29,7 +29,7 @@ type VerificationMode = 'providerId' | 'jsonConfig' | 'url';
 
 export default function App() {
     const [verificationMethod, setVerificationMethod] = useState<VerificationMode>('providerId');
-    const [inputText, setInputText] = useState('6d3f6753-7ee6-49ee-a545-62f1b1822ae5');
+    const [inputText, setInputText] = useState('example');
```

ESLint

[error] Delete `··`

(prettier/prettier)